docs/ui/automations.md

```diff
@@ -70,14 +89,105 @@ Similarly, if a trigger fires on work queue health and the action is to pause an
 
 Prefect tries to infer the relevant event whenever possible, but sometimes one does not exist.
 
-## Details
+### Details
 
-Finally, specify a name and, optionally, a description for the automation.
+Specify a name and, optionally, a description for the automation.
 
 ![Configuring details for an automation in Prefect Cloud.](/img/ui/automations-details.png)
 
-## Coming soon
+## Automation notifications
+
+Notifications enable you to set up automation actions that send a message. 
+
+Automation notifications support sending notifications via any predefined block that is capable of and configured to send a message. That includes, for example:
+
+- Slack message to a channel
+- Microsoft Teams message to a channel
+- Email to a configured email address
+
+![Configuring notifications for an automation in Prefect Cloud.](/img/ui/automations-notifications.png)
+
+!!! note "Notification blocks must be pre-configured"
+    Notification blocks must be pre-configured prior to creating a notification action. Any existing blocks capable of sending messages will be shown in the block drop-down list.
+
+    The **Add +** button cancels the current automation creation process and enables configuration a notification block.
+
+## Templating notifications with Jinja
+
+The notification body can include templated variables using [Jinja](https://palletsprojects.com/p/jinja/) syntax. Templated variable enable you to include details relevant to automation trigger, such as a flow or queue name. 
+
+Jinja templated variable syntax wraps the variable name in double curly brackets, like `{{ variable }}`.
+
+You can access properties of the underlying flow run objects including:
+
+- [flow_run](/api-ref/orion/schemas/core/#prefect.orion.schemas.core.FlowRun)
+- [flow](/api-ref/orion/schemas/core/#prefect.orion.schemas.core.Flow)
+- [deployment](/api-ref/orion/schemas/core/#prefect.orion.schemas.core.Deployment)
+- [work_queue](/api-ref/orion/schemas/core/#prefect.orion.schemas.core.WorkQueue)
+
+In addition to its native properites, each object includes an `id` along with `created` and `updated` timestamps. 
+
+The `flow_run|ui_url` token returns the URL for viewing the flow run in Prefect Cloud.
```

Comment:
This sentence seems like it was orphaned, I don't see `ui_url` actually being used in any of the examples.

Reply:
Good note. I wrote it this way intentionally. It may not be expressed well. We'll revisit this whole section once we get some feedback on where users are experiencing frictions.